server/test/client.jl

```diff
@@ -33,7 +33,7 @@ end
 
 @testset "model information functions" begin
     @test request((fn = "get_component_name",)) == Dict("component_name" => "sbm")
-    @test request((fn = "get_input_item_count",)) == Dict("input_item_count" => 181)
+    @test request((fn = "get_input_item_count",)) == Dict("input_item_count" => 180)
     @test request((fn = "get_output_item_count",)) == Dict("output_item_count" => 181)
```

Comment:
I feel like checking the precise numbers here and in the Wflow BMI tests doesn't help much and needs a lot of maintenance. Should we just check if it is more than a hundred or something like that?

Reply:
Good point! We have decided to keep this test as is, but made the tests below a bit more flexible. Maybe we want to reconsider this in the future, but for now we are fine with the current implementation.